todo.todo

```diff
@@ -1,62 +0,0 @@
-☐ practice git commands for rebase and solving merge conflicts
```

Comment:
let's keep that as a record 💃  - just for presenting the progress - a nice thing to have 😃